eng/WpfArcadeSdk/tools/RuntimeFrameworkReference.targets

```diff
@@ -3,6 +3,15 @@
     Explicitly import Microsoft.NETCore.Platforms to prevent using the one from the SDK, which
     usually lags behind the version that WPF gets from CoreFx packages
   -->
+
+  <PropertyGroup>
+    <UpdatedTargetingPackVersion Condition="'$(MicrosoftNETCoreAppRefVersion)'!=''
+                                            And '$(NoTargets)'!='true'
+                                            And '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+                                            And $([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '3.0'))
+                                            And '$(MSBuildProjectExtension)'!='.vcxproj'">$(MicrosoftNETCoreAppRefVersion)</UpdatedTargetingPackVersion>
+  </PropertyGroup>
```

Comment:
You should be able to revert all the changes in this file and merge latest main into your branch. My PR that fixed this got merged yesterday.